charts/consul/test/unit/ingress-gateways-deployment.bats

```diff
@@ -1504,3 +1504,35 @@ key2: value2' \
   [ "${actualTemplateFoo}" = "bar" ]
   [ "${actualTemplateBaz}" = "qux" ]
 }
+
+#--------------------------------------------------------------------
+# logLevel
+
+@test "ingressGateways/Deployment: use global.logLevel by default" {
+  cd `chart_dir`
+  local cmd=$(helm template \
+      -s templates/ingress-gateways-deployment.yaml  \
+      --set 'ingressGateways.enabled=true' \
+      --set 'connectInject.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.initContainers[0].command' | tee /dev/stderr)
```

Comment:
You'll want to test both containers when you come up with a solution that let's you pass `trace` to dataplane but not the initContainer.

This applies to the wider set of deployments w/ dataplane containers.

Reply:
+1

Reply:
Added additional tests